spec/system/page_objects/modals/reject_reason_reviewable.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Modals
+    class RejectReasonReviewable < PageObjects::Pages::Base
+      def modal
+        find(".reject-reason-reviewable-modal")
+      end
+
+      def select_send_rejection_email_checkbox
+        modal.check("Send rejection email")
```

Comment:
Can we please use the translation `I18n` here? Otherwise this spec will break if the translation changes.

Reply:
Sounds like you have a solution, but for future reference I believe we can access 'client' translations on the server by prefixing `js.` like this:

```
I18n.t("js.some_key")
```

Reply:
Yeah class is fine but what David says works dandy too :+1: